src/client/src/ui/spotTile/components/PriceControls.tsx

```diff
@@ -17,33 +19,117 @@ interface Props {
   priceData: SpotPriceTick
   executeTrade: (direction: Direction, rawSpotRate: number) => void
   disabled: boolean
+  rfqState: RfqState
 }
 
-const PriceControls: React.FC<Props> = ({ currencyPair, priceData, executeTrade, disabled }) => {
+const PriceButtonDisabledPlaceholder = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-flow: column;
+  border: 2px solid ${({ theme }) => theme.core.darkBackground};
+  border-radius: 3px;
+  font-size: 10px;
+  transition: background-color 0.2s ease;
+  height: 58px;
+  line-height: normal;
+  min-width: 126.13px;
```

Comment:
Good question 🤔 ... The PriceControls are always that length (when containing prices), because they are sized according to the content + padding which never changes (on any given viewport size).

But for the PriceControls to have the exact same size while showing different messages (not Prices) I had to hardcode this.

Any suggestion for a better solution is welcomed.

One thing sure, it definitely would be better to have the `PriceButton` and `PriceButtonDisabledPlaceholder` components use the same "sizing strategy", ie. both use fixed size for the button, no padding, css grid, etc.

What do you think?

I know @bhavesh-desai-scratch wanted us to look into the entire Tile layout and migrate it to CSS grid (SpotTile, AnalyticsTile, etc)

![Screen Shot 2019-04-03 at 1 00 13 PM](https://user-images.githubusercontent.com/1565793/55497960-75f58180-5610-11e9-8b1e-c0df5ac876cc.png)